### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/gnucash/gnucash/) for GnuCash/Program.

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -17507,9 +17512,8 @@ msgid "Case insensitive searching is available on 'Account Full Name'."
 msgstr ""
 
 #: gnucash/gtkbuilder/dialog-find-account.glade:230
-#, fuzzy
 msgid "_Search"
-msgstr " Recherche "
+msgstr "Recherche"
```
Comment (fellen) [inline po/fr.po:17516]: @ImagesLD2503 please read "_" (underline) in https://wiki.gnucash.org/wiki/Translation#Special_characters_and_other_tips

**Comment by fellen** [comment]
@avma 
```
msgfmt -c --check-accelerators="_" --statistics he.po
he.po:3513: msgstr lacks the keyboard accelerator mark '_'
he.po:4582: msgstr lacks the keyboard accelerator mark '_'
msgfmt: found 2 fatal errors
5547 translated messages.
```
@ImagesLD2503
```
msgfmt -c --check-accelerators="_" --statistics fr.po
fr.po:4469: msgstr lacks the keyboard accelerator mark '_'
fr.po:17516: msgstr lacks the keyboard accelerator mark '_'
fr.po:18591: msgstr lacks the keyboard accelerator mark '_'
fr.po:21528: msgstr lacks the keyboard accelerator mark '_'
fr.po:23034: number of format specifications in 'msgid_plural' and 'msgstr[1]' does not match
fr.po:23044: number of format specifications in 'msgid_plural' and 'msgstr[1]' does not match
msgfmt: found 6 fatal errors
3861 translated messages, 1159 fuzzy translations, 527 untranslated messages.

```
@yyyjajp 
```
msgfmt -c --check-accelerators="_" --statistics ja.po
ja.po:6632: msgstr lacks the keyboard accelerator mark '_'
ja.po:16347: msgstr lacks the keyboard accelerator mark '_'
ja.po:18736: msgstr lacks the keyboard accelerator mark '_'
ja.po:18951: msgstr lacks the keyboard accelerator mark '_'
ja.po:20280: msgstr lacks the keyboard accelerator mark '_'
ja.po:29940: msgstr lacks the keyboard accelerator mark '_'
msgfmt: found 6 fatal errors
5150 translated messages, 158 fuzzy translations, 239 untranslated messages.
```
Please fix this errors.

And thanks foryour contribution!

**Comment by yyyjajp** [comment]
@fellen 
```
ja.po:20278:#: gnucash/import-export/csv-exp/csv-tree-export.c:155
ja.po:20279:msgid "full_name"
ja.po:20280:msgstr "フルネーム"
```
which is not a keyboard accelerator but a column name.

**Comment by fellen** [comment]
@yyyjajp Thanks, I asked the author for an improvement: https://lists.gnucash.org/pipermail/gnucash-devel/2021-January/045659.html

**Pull request merged**